community-rust-plugin/src/main/java/org/elegoff/plugins/communityrust/xunit/XUnitSensor.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * Community Rust Plugin
+ * Copyright (C) 2021 Eric Le Goff
+ * mailto:community-rust AT pm DOT me
+ * http://github.com/elegoff/sonar-rust
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.elegoff.plugins.communityrust.xunit;
+
+
+import org.elegoff.plugins.communityrust.language.RustLanguage;
+import org.sonar.api.batch.fs.InputComponent;
+import org.sonar.api.batch.fs.InputFile;
+import org.sonar.api.batch.sensor.Sensor;
+import org.sonar.api.batch.sensor.SensorContext;
+import org.sonar.api.batch.sensor.SensorDescriptor;
+import org.sonar.api.config.Configuration;
+import org.sonar.api.measures.CoreMetrics;
+import org.sonar.api.measures.Metric;
+import org.sonar.api.utils.log.Logger;
+import org.sonar.api.utils.log.Loggers;
+import org.sonarsource.analyzer.commons.FileProvider;
+
+import javax.xml.stream.XMLStreamException;
+import java.io.File;
+import java.util.List;
+
+
+public class XUnitSensor implements Sensor {
+    public static final String REPORT_PATH_KEY = "sonar.test.reportPath";
```

Comment:
I would change `sonar.test.reportPath` into `sonar.rust.test.reportPath` to make it plugin specific

Reply:
Change made in 22e6af84a8ac085ecbaa9648e71c83dc9ce0a10a